app/controllers/idv/please_call_controller.rb

```diff
@@ -22,5 +23,10 @@ def confirm_fraud_pending
         redirect_to account_url
       end
     end
+
+    def in_person_proofing?
```

Comment:
this is not a big thing but i'm wondering about naming here. This is checking both that the user went through ipp and that the status is passed.  Is there maybe some naming that could capture both? not blocking at all though